dist/rapidast/README.md

```diff
@@ -0,0 +1,26 @@
+# RapidAST Security Scanning
+
+Run automated security scans against OpenShift APIs using RapidAST container.
+
+## Quick Start
+
+```bash
+# Pull the container
+podman pull quay.io/redhatproductsecurity/rapidast:latest
+
+# Create results directory
+mkdir -p results
+
+# Run security scan
+podman run --rm \
+  -v $(pwd)/rapidast-config.yaml:/tmp/config.yaml \
+  -v $(pwd)/results:/opt/rapidast/results \
+  quay.io/redhatproductsecurity/rapidast:latest \
+  --config /tmp/config.yaml
```

Comment:
Note: We can omit the `--config` flag if we utilize the default configuration path `/opt/rapidast/config/config.yaml` for the mount. Noting this in case you find it helpful.